src/Parser/DemonHunter/Vengeance/Modules/Abilities.js

```diff
@@ -170,6 +183,13 @@ class Abilities extends CoreAbilities {
         cooldown: 20,
         charges: 2,
       },
+      {
+        spell: SPELLS.SIGIL_OF_SILENCE,
+        category: Abilities.SPELL_CATEGORIES.UTILITY,
+        enabled: combatant.hasTalent(SPELLS.QUICKENED_SIGILS_TALENT.id),
+        cooldown: 48,
```

Comment:
This cooldown looks to be incorrect, the tooltip says it's 60 seconds.

Reply:
Instead of having 2 entries for each Sigil, it's possible to use 1 and change the CD based on whether the talent is chosen - like this example from resto shaman (CD with : without talent).
> cooldown: combatant.hasTalent(SPELLS.GRACEFUL_SPIRIT_TALENT.id) ? 60 : 120,